values.yaml

```diff
@@ -121,11 +116,11 @@ auth:
     authExpiration: 12 hours
 
   oidc:
-    enabled: false
-    discoveryUrl:
-    clientId:
-    clientSecret:
-    claimIdentifyingUser: email
+    enabled: true
```

Comment:
the chart defaults should not have a dependency on an auth system, should not this be false by default?